devtools/gradle/src/main/java/io/quarkus/gradle/tasks/QuarkusAddExtension.java

```diff
@@ -31,6 +36,17 @@ public void setExtensionsToAdd(List<String> extensionsToAdd) {
         return extensionsToAdd;
     }
 
+    @Optional
+    @Input
+    public String getRegistry() {
+        return registry;
+    }
+
+    @Option(description = "The extension registry URL to be used", option = "registry")
```

Comment:
where did we end up on having list of registries to support layering of registries?

Reply:
The API is ready for this, in my Quarkus CLI implementation I read a list of URLs from the `application.yaml`. I could make it a list here though

Reply:
+1 for that.